pytest.ini

```diff
@@ -1,4 +1,5 @@
 [pytest]
+python_files = test*.py
```

Comment:
The reason the other line is split up is because it is taking multiple values. For line 2, we only are assigning one value so we don't need a new line.

Reply:
sounds good